### ISSUE TITLE
[A11y]Tooltip provided for links on Front page are not provided for keyboard user

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8493]
Link: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1296225

​## Environment Details:
Application Name: NuGet Service
#URL: https://www.nuget.org
Operating System: Windows 10
Browser Version: Chrome Version 85.0.4183.121 (Official Build) (64-bit),
Chromium Edge Version 87.0.637.0 (Official build) dev (64-bit)

## Repro Steps:​
 Hit the URL " https://www.nuget.org" and login with appropriate credentials.
Tab Navigate to "Sign in" control and hit enter.
 Tab Navigate to "Sign in with Microsoft" control and hit enter.
Tab navigate to learn more link and observe tooltip is provided for keyboard user or not.
## Actual:​
Tooltip provided for learn more link is not provided for keyboard user as its visible on mouse hover over the link.
## Expected:​
Tooltip should be provided for learn more link is not provided for keyboard user as its visible on mouse hover over the link.
​## User Impact:​
If tooltip is not actionable using keyboard then keyboard user will some information that provided in the tooltip.

### Notes
This appears to be in references to the links on the front page (from the video attached in the provided link). We should evaluate how difficult it is to fix this since I believe that the information provided as text below would be sufficient alternate source of the same information.

**Comment by drewgillies** [comment]
Resolving this without making a change--the screenreader reads similar text to the tooltip, and for a keyboard user, the text below the link is equivalent (and more detailed).
